### ISSUE TITLE
Option for graceful stop of server

### ISSUE THREAD

**Issue by motz0815** [jpenilla/run-paper#17]
It would be really useful to be able to just run the runServer task again inside of IntelliJ, and that the server then closes gracefully, like if you Ctrl-C inside of the console. This can be done by sending a SIGINT to the JVM, so it has to be possible somehow.

Maybe also a "config" option for people that (don't) need this functionality. Though the maintainer would have to decide what's the default, I'd be happy with both.

**Comment by jpenilla** [comment]
I don't think this is feasible without an intellij plugin

**Comment by DenWav** [comment]
Doesn't IntelliJ default to "stop and re-run" when you click run while it's already running?

**Comment by Leguan16** [comment]
Yeah but it stops with an error and doesn't properly shut down the server so you will never execute `onDisable` with the plugin (unless you manually type stop in the console)
![image](https://github.com/jpenilla/run-task/assets/59799222/58810e39-27b0-4e85-8b27-060e5dd1d3e2)

**Comment by Ifiht** [comment]
FYI I had this problem, made a plugin for it: https://github.com/Ifiht/AutoStop

I just include it in the plugin downloads as a custom run-task for automated builds, so it doesn't fail things. Workaround until something better happens.
